django_project/settings.py

```diff
@@ -151,3 +151,12 @@
 
 # Minimal string length considered similarity
 MIN_SIMILARITY_LENGTH = 10
+
+# Email settings, service provider SendGrid
+SENDGRID_API_KEY = env("SENDGRID_API_KEY")
+
+EMAIL_HOST = 'smtp.sendgrid.net'
+EMAIL_HOST_USER = 'apikey' # this is exactly the value 'apikey'
+EMAIL_HOST_PASSWORD = SENDGRID_API_KEY
```

Comment:
We could just leave it like that: `EMAIL_HOST_PASSWORD = env("SENDGRID_API_KEY")`. If we wanted it more modular in the future, we would need to refactor making host or user modular.